src/components/TransactionHistory/TransactionHistory.jsx

```diff
@@ -109,6 +120,26 @@ const TransactionHistory = (props) => {
   );
 };
 
+function setKeyDisplay(key) {
+  let keyToDisplay = key;
+  if (key === "cumulativeGasUsed") {
+    keyToDisplay = "cumulative gas used";
+  }
+  else if (key === "gasUsed") {
+    keyToDisplay = "gas used";
+  }
+  else if (key === "transactionHash") {
+    keyToDisplay = "transaction hash";
+  }
+  else if (key === "effectiveGasPrice") {
+    keyToDisplay = "gas price";
+  }
+  else if (key === "contractAddress") {
+    keyToDisplay = "contract address";
+  }
+  return keyToDisplay;
```

Comment:
maybe cleaner way would be to make another object like 
```
keyToDisplay  = {
"cumulativeGasUsed": "cumulative gas used",
"gasUsed": "gas used", 
...etc
}
```
and then use keyToDisplay[key] instead of calling a func